keyboards/cannonkeys/serenity/info.json

```
@@ -0,0 +1,147 @@
+{
+    "keyboard_name": "Serenity",
+    "usb": {
+        "vid": "0xCA04",
+        "pid": "0x0017",
+        "device_version": "0.0.1"
+    },
+    "url": "https://cannonkeys.com",
+    "maintainer": "awkannan",
+    "bootloader": "stm32-dfu",
+    "processor": "STM32F072",
+    "diode_direction": "COL2ROW",
+    "features": {
+        "bootmagic": true,
+        "command": false,
+        "console": false,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true,
+        "backlight": true,
+        "rgblight": true
+    },
+    "debounce": 5,
+    "matrix_pins": {
+        "cols": ["B1", "B2", "B10", "B11", "B12", "B14", "A8", "A9", "A10", "A3", "B0", "A2", "A1", "A7", "A0", "B4", "B6", "B7"],
+        "rows": ["A15", "B3", "B5", "A4", "A5", "F1"]
+    },
+    "backlight": {
+        "breathing": true,
+        "breathing_period": 5,
+        "levels": 15,
+        "pin": "A6"
+    },
+    "rgblight": {
+        "led_count": 12,
+        "pin": "B15",
+        "hue_steps": 24,
+        "saturation_steps": 16,
+        "brightness_steps": 16,
+        "animations": {
+            "all": true
```

Could you explicitly define which animations are enabled? 
```suggestion
            "alternating": true,
            "breathing": true,
            "christmas": true,
            "knight": true,
            "rainbow_mood": true,
            "rainbow_swirl": true,
            "rgb_test": true,
            "snake": true,
            "static_gradient": true,
            "twinkle": true
```

---

I can do this - but as a note, wouldn't it be best to remove the "all" option if you don't want anyone to use it?

---

Yeah, that's the plan for the next cycle.